README.md

```diff
@@ -51,12 +66,43 @@ module "power-infrastructure" {
   perform_proxy_client_setup  = var.perform_proxy_client_setup
 }
 ```
+
+<!-- PERMISSIONS REQUIRED TO RUN MODULE
+## Required IAM access policies
+
+If this module requires permissions, uncomment the following block and update
+the sample permissions, following the format.
+Replace the sample Account and IBM Cloud service names and roles with the
+information in the console at
+Manage > Access (IAM) > Access groups > Access policies.
+-->
+
+<!--
+You need the following permissions to run this module.
+
+- Account Management
+    - **Sample Account Service** service
+        - `Editor` platform access
+        - `Manager` service access
+    - IAM Services
+        - **Sample Cloud Service** service
+            - `Administrator` platform access
+-->
+
+<!-- NO PERMISSIONS FOR MODULE
+If no permissions are required for the module, uncomment the following
+statement instead the previous block.
+-->
+
+<!-- No permissions are needed to run this module.-->
+<!-- END MODULE HOOK -->
+
 <!-- BEGIN EXAMPLES HOOK -->
 ## Examples
 
-- [ Basic PowerVS Infrastructure Module Example](examples/basic)
-- [ Catalog PowerVS Infrastructure Module Example](examples/ibm-catalog/standard-solution)
-- [ Standard PowerVS Infrastructure Module Example](examples/standard)
+- [ Basic example for Power infrastructure for regulated industries](examples/basic)
+- [ Catalog example for Power infrastructure for regulated industries](examples/ibm-catalog/standard-solution)
```

Comment:
I'd add "IBM Cloud Catalog" here - as people looking on github may not be aware of the IBM Cloud Catalog capability

Reply:
It would need to be updated in the title of `examples/ibm-catalog/standard-solution/README.md` and the pre commit hook would then automatically update the main readme

Reply:
I'll push an update to the example readme